ansible_collections/arista/avd/plugins/module_utils/eos_designs_facts.py

```diff
@@ -663,6 +663,24 @@ def spanning_tree_root_super(self):
             return get(self._switch_data_combined, "spanning_tree_root_super")
         return None
 
+    @cached_property
+    def underlay_multicast(self):
+        return get(self._hostvars, "underlay_multicast")
+
+    @cached_property
+    def evpn_multicast(self):
+        if get(self._hostvars, "evpn_multicast") is True and self.vtep is True:
+            if not (
+                self.underlay_multicast is True
+                and self.igmp_snooping_enabled is not False
+            ):
+                raise AristaAvdError(
+                    "'evpn_multicast: True' is only supported in "
+                    "combination with 'underlay_multicast: True' and 'igmp_snooping_enabled : True'"
+                )
+            return get(self._hostvars, "evpn_multicast", default=False)
```

Comment:
could we add a check that if mlag we also check for hostvars `overlay_rd_type.admin_subfield` is either "overlay_lookback", unset (default="overlay_loopback") or that the value is different that the value of the mlag peer (same key). You can find the mlag peer's value by accessing `avd_switch_facts` with the `get` function. See peer_mlag_id for an example.

Reply:
Yes, will add this check!

Reply:
Completed